patch/series

```diff
@@ -189,8 +189,29 @@ cisco-hwmon-pmbus_core-pec-support-check.patch
 0003-psample-define-the-macro-PSAMPLE_MD_EXTENDED_ATTR.patch
 
 #
-# Marvell platform patches for 4.19
+# Marvell platform patches for 5.10
 armhf_secondary_boot_online.patch
```

Comment:
has this patch being upstreamed?

Reply:
> has this patch being upstreamed?

Please find upstreamed patches in the commit description and link has been given to them.

Reply:
Snippet from commits, referencing the upstream commit information.
![image](https://github.com/sonic-net/sonic-linux-kernel/assets/92989231/c2ea26aa-46c6-40e2-89aa-ba8c0eaeb67a)